target/linux/ath79/image/mikrotik.mk

```diff
@@ -48,6 +48,16 @@ define Device/mikrotik_routerboard-922uags-5hpacd
 endef
 TARGET_DEVICES += mikrotik_routerboard-922uags-5hpacd
 
+define Device/mikrotik_routerboard-951ui-2hnd
+  $(Device/mikrotik_nand)
+  SOC := ar9344
+  DEVICE_MODEL := RouterBOARD 951Ui-2HnD
+  DEVICE_PACKAGES += kmod-usb-ohci kmod-usb2
+  SUPPORTED_DEVICES += rb-951ui-2hnd
+  IMAGE/sysupgrade.bin = append-kernel | sysupgrade-tar | append-metadata
```

Comment:
Very minor nit: I would instead add a `mikrotik_nandyafut` (or somesuch) common recipe in `common-mikrotik.mk`, and use that as `$(Device/mikrotik_nandyafut)` instead of `$(Device/mikrotik_nand)`.

This will avoid cluttering every device that is moved to this system and will make it easy to spot the ones that have been switched over and/or mass switch the remaining ones over.

Reply:
Ah, yes, this is definitely a good idea, thanks.  I guess the `IMAGE/sysupgrade.bin` line you commented on is simply a leftover from my initial attempts to get things working that I never touched again because I did not need to.  But the approach you suggested is clearly superior in the general case.

While you pointed this out in this PR, it seems to me that adding a new `Device/...` definition to `target/linux/ath79/image/common-mikrotik.mk` should rather happen in #12225 and this PR should only use that definition.  I will assume that approach for the time being, but feel free to let me know if you disagree.

Reply:
I added the device template suggested above in dacf09232647d2742fedc2f59f96d93e7e7732c2, which is part of #12225.  ee0b5862e30b55f03e7f76d47e1a83ae50c6f719, i.e. the current version of this PR, uses that template.

I am going to resolve this discussion, but obviously please let me know if you have any further comments.  Thanks!

Reply:
This was approached in a different after all, see [elsewhere](https://github.com/openwrt/openwrt/pull/12225#discussion_r1154366821).